Sources/Testing/Support/Additions/CommandLineAdditions.swift

```diff
@@ -16,16 +16,15 @@ extension CommandLine {
     get throws {
 #if os(macOS)
       var result: String?
-      var bufferCount = UInt32(1024)
+      var bufferCount = UInt32(PATH_MAX)
       while result == nil {
-        result = withUnsafeTemporaryAllocation(of: CChar.self, capacity: Int(bufferCount)) { buffer in
```

Comment:
Let's consistently _set_ `result` in the buffer's scope rather than returning it. This saves us a `return nil` line and will be easier to translate if/when we get non-escaping buffers.